.gitlab-ci.yml

```
@@ -119,7 +119,7 @@ before_script:
     # and https://github.com/coq/coq/pull/11916#issuecomment-609977375
     - ulimit -s 16384
     - set -e
-    - make -f Makefile.dune world
+    - dune build -pcoq,coqide-server,coqide @install
```

```suggestion
    - dune build -p coq,coqide-server,coqide @install
```
not having a space here looks terrible imo

---

Done.